CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 ## [1.2.2]
```

Comment:
In order to make this work we need to add this link definition at the bottom of the file like:

```
[1.2.1]: https://github.com/esri/calcite-web/compare/v1.2.1...v1.2.2
```

That turns the header into a link to the diff